Office 365/o365/tests/security_compliance_alert_3.json

```
@@ -0,0 +1,74 @@
+    "sekoiaio": {
+      "intake": {
+        "dialect": "office 365",
+        "dialect_uuid": "caa13404-9243-493b-943e-9848cadb1f99"
+      }
+    }
+  },
+  "expected": {
+    "message": "{\"CreationTime\": \"2024-04-04T08:09:07\", \"Id\": \"9bb5e4b2-febd-4f1a-93bb-1380f7868320\", \"Operation\": \"AlertEntityGenerated\", \"OrganizationId\": \"9eb6ae86-0209-4328-856e-3f7971a6c654\", \"RecordType\": 40, \"ResultStatus\": \"Succeeded\", \"UserKey\": \"SecurityComplianceAlerts\", \"UserType\": 4, \"Version\": 1, \"Workload\": \"SecurityComplianceCenter\", \"ObjectId\": \"4dde17d2-635a-498b-914e-73898a280433\", \"UserId\": \"SecurityComplianceAlerts\", \"AlertEntityId\": \"a1bba0c6-d8d0-4710-8fc3-6085b3ac79fc\", \"AlertId\": \"178fa649-642f-4d41-943c-451e2266f4a7\", \"AlertLinks\": [{\"AlertLinkHref\": \"\"}], \"AlertType\": \"Custom\", \"Category\": \"MailFlow\", \"Comments\": \"New alert\", \"Data\": \"{\\\"etype\\\":\\\"MalwareFamily\\\",\\\"at\\\":\\\"2024-04-04T08:07:58.0000000Z\\\",\\\"md\\\":\\\"2024-04-04T08:07:58.0000000Z\\\",\\\"sip\\\":\\\"217.182.111.68\\\",\\\"ms\\\":\\\"[MARKETING] Free movies for you\\\",\\\"imsgid\\\":\\\"<f2231b61bc5cbb95893c5b62281e4f4c@spam.com>\\\",\\\"dm\\\":\\\"CrossOrgSpoof\\\",\\\"eid\\\":\\\"58fe2050-27be-431f-a035-2cba142cb021-15828634862951700795-1\\\",\\\"aii\\\":\\\"58fe2050-27be-431f-a035-2cba142cb021\\\",\\\"thn\\\":\\\"Phish, Malicious\\\",\\\"ts\\\":\\\"2024-04-04T08:06:58.0000000Z\\\",\\\"te\\\":\\\"2024-04-04T08:08:58.0000000Z\\\",\\\"fvs\\\":\\\"Filters\\\",\\\"tpt\\\":\\\"AntiPhishPolicy\\\",\\\"tpid\\\":\\\"b3caa8a3-c6de-4d06-b2a4-8968c6d1f2d5\\\",\\\"tid\\\":\\\"7fca7e7b-11e7-48c7-b345-1145a7a3a918\\\",\\\"tht\\\":\\\"Phish, Malicious\\\",\\\"trc\\\":\\\"john.doe@example.org\\\",\\\"tsd\\\":\\\"communications@spam.com\\\",\\\"tdc\\\":\\\"1\\\",\\\"cpid\\\":null,\\\"lon\\\":\\\"Protection\\\"}\", \"EntityType\": \"MalwareFamily\", \"Name\": \"Phishing detected\", \"PolicyId\": \"40663ca5-73da-43ca-b0be-af8b54987339\", \"Severity\": \"Low\", \"Source\": \"Office 365 Security & Compliance\", \"Status\": \"Active\"}",
+    "event": {
+      "action": "AlertEntityGenerated",
+      "category": [
+        "intrusion_detection"
+      ],
+      "code": "40",
+      "kind": "alert",
+      "outcome": "success",
+      "type": [
+        "info"
+      ]
+    },
+    "@timestamp": "2024-04-04T08:09:07Z",
+    "action": {
+      "id": 40,
+      "name": "AlertEntityGenerated",
+      "outcome": "success",
+      "target": "user"
+    },
+    "email": {
+      "message_id": "<f2231b61bc5cbb95893c5b62281e4f4c@spam.com>",
```

message_id must be without the < and >
```suggestion
      "message_id": "f2231b61bc5cbb95893c5b62281e4f4c@spam.com",
```